lib/Model.test.js

```diff
@@ -0,0 +1,187 @@
+const Schema = require('./Schema');
+const Model = require('./Model');
+
+
+
+describe('Model class', () => {
```

Comment:
Tests look great but you want an afterAll() that will clear out all your saved dogs